src/config/webpack.common.js

```diff
@@ -17,16 +17,22 @@ const copies = {
 const { eslintConfig, main: entryPoint } = require(resolveExtensionPath("package.json"));
 const eslintEnabled = eslintConfig || fs.readdirSync(extensionPath).find(f => f.startsWith(".eslintrc"));
 const jsLoaders = [{
-    loader: require.resolve("babel-loader"),
+    loader: "babel-loader",
     options: {
         presets: [
-            [require.resolve("@babel/preset-env"), {
-                targets: {
-                    chrome: 45,
-                    safari: "9.1",
-                    firefox: 45
+            [
+                "@babel/preset-env",
+                {
+                    useBuiltIns: "usage",
+                    corejs: 3,
+                    modules: false, // Should be false to run tree shaking. See: https://webpack.js.org/guides/tree-shaking/
+                    targets: {
+                        chrome: 45,
+                        safari: "9.1",
+                        firefox: 45
```

Comment:
Let's update the supported versions with respect to [this article](https://support.zeplin.io/en/articles/1090159-supported-versions):

```suggestion
                        chrome: 62,
                        safari: 11,
                        firefox: 59,
                        edge: 15
```

PS: For Safari, we use the version that comes as default with the oldest macOS version Zeplin supports (macOS 10.13).